x/stakeibc/keeper/msg_server_register_host_zone.go

```diff
@@ -92,12 +92,13 @@ func (k msgServer) RegisterHostZone(goCtx context.Context, msg *types.MsgRegiste
 		HostDenom:         msg.HostDenom,
 		TransferChannelId: msg.TransferChannelId,
 		// Start exchange rate at 1 upon registration
-		RedemptionRate:     sdk.NewDec(1),
-		LastRedemptionRate: sdk.NewDec(1),
-		UnbondingPeriod:    msg.UnbondingPeriod,
-		DepositAddress:     depositAddress.String(),
-		MinRedemptionRate:  msg.MinRedemptionRate,
-		MaxRedemptionRate:  msg.MaxRedemptionRate,
+		RedemptionRate:        sdk.NewDec(1),
+		LastRedemptionRate:    sdk.NewDec(1),
+		UnbondingPeriod:       msg.UnbondingPeriod,
+		DepositAddress:        depositAddress.String(),
+		MinRedemptionRate:     msg.MinRedemptionRate,
+		MaxRedemptionRate:     msg.MaxRedemptionRate,
+		LsmLiquidStakeEnabled: msg.LsmLiquidStakeEnabled,
```

Comment:
We can't call this function if the host zone is already registered, so it is only on the startup of the chain correct? Meaning the chain has to be restarted to update the LSM status for any other chain? Is this the desired behavior or should it be more like a param we can update without needing to restart our chain?  Presumably once a 3rd party chain goes from false to true, they are not going in the other direction ever by removing the LSM module...

Reply:
We should enable it for the hub in the software upgrade that adds LSM. For other host zones, it probably makes the most sense to expose a governance function to toggle it